### ISSUE TITLE
Backport extensions from other Scratch mods (e.g. Adacraft, PenguinMod, etc.) to TurboWarp

### ISSUE THREAD

**Issue by HBDeveloper11** [TurboWarp/extensions#1047]
The title is self-explanatory. After looking at some other Scratch mods, I've noticed that there were extensions that I really liked that needed to be in TurboWarp. But the thing is, that some of those extensions might not comply with the TurboWarp extension guidelines, so maybe you can only add the extensions that comply with the extension guidelines. It would be really nice if this actually happened, because we would have more extensions, and more features.

**Comment by CubesterYT** [comment]
You could try porting them yourself, nothing's stopping you because they're open source.

**Comment by HBDeveloper11** [comment]
Good idea. But I might need to tweak the extensions so that they can work properly in TurboWarp.

**Comment by SurvExe1Pc** [comment]
i love this idea (if this is replying to the latest message its cause idk how to do it right in emai)
but yeah i would love to see back ports, I would love to see you get some working ones.
________________________________

**Comment by HBDeveloper11** [comment]
I will attempt to back-port these extensions later today, and I'll give you my results once it's successful. (this is my alt account, my main account is DevelopCMD)

**Comment by SurvExe1Pc** [comment]
oh nice (btw if you see a dupe msg please mark it as dupe, its a bug w/ outlook, thx)
________________________________

**Comment by HBDeveloper11** [comment]
this probably isnt gonna work out
